### PR TITLE
Css layout classes- David Gonzalez

### PR DESCRIPTION
**Pull request by David-Gonzalez11**
Screen Shot 2022-05-12 at 2 47 16 PM

### CODE REVIEW

**Comment by David-Gonzalez11** [comment]
I used loreum ipsum if that is okay

File: css-layout-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  <head>
+    <title>
+      CSS Layout classes
+    </title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <h1>Choose your character!</h1>
```
Comment (Zhiwutian) [inline css-layout-classes/index.html:12]: you need a column here for this content. The column should be full width

File: css-layout-classes/index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        </div>
+        <div class="column-one-third"></div>
+        <img src="images/dk.png">
+      </div>
+
+      <div class="row">
+        <div class="column-one-third mario">
+          <img src="images/mario.png" class="mario">
+        </div>
+        <div class="column-two-thirds">
```
Comment (Zhiwutian) [inline css-layout-classes/index.html:56]: make sure you have a css selector and styling for this class

File: css-layout-classes/index.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      CSS Layout classes
+    </title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <h1>Choose your character!</h1>
+      </div>
+
```
Comment (Zhiwutian) [inline css-layout-classes/index.html:14]: make sure to remove extra whitespace like this between elements

File: css-layout-classes/index.html [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+      <div class="row">
+        <div class="column-one-third">
+          <img src="images/link.png">
```
Comment (Zhiwutian) [inline css-layout-classes/index.html:17]: make sure to add alt text to all images

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
on its way, a few updates are needed

File: css-layout-classes/styles.css [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+.mario {
+  width: 100%;
+}
+
+.column-full {
+  width: 100%;
+}
+
+.column-two-third {
```
Comment (Zhiwutian) [inline css-layout-classes/styles.css:32]: make sure the width matches the class name, 2/3 should be 2/3, not 3/4

File: css-layout-classes/styles.css [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-one-fourth {
+  width: calc(100% / 4);
+  display: flex;
```
Comment (Zhiwutian) [inline css-layout-classes/styles.css:13]: you can remove display flex here as you don't need to control the child of the column class in this instance

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
almost there! Make sure to update the pull request with a new screenshot after changes are made

**Comment by David-Gonzalez11** [comment]
mariooo

File: css-layout-classes/index.html [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+          <img src="images/ness.png" alt="ness">
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-two-thirds">
+          <h1>Donkey Kong</h1>
+          <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Sed accusantium deserunt ex quia iste eaque rem sit veritatis. Ducimus facilis et vel ad ipsum modi quibusdam recusandae, eveniet quia veniam.</p>
+        </div>
+        <div class="column-one-third"></div>
+        <img src="images/dk.png" alt="Donkey Kong">
```
Comment (Zhiwutian) [inline css-layout-classes/index.html:48]: make sure Donkey Kong is within the column class!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one tiny update!

**Comment by David-Gonzalez11** [comment]
Screen Shot 2022-05-12 at 3 16 12 PM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**